ceaos/sensors/PiCamera.py

```diff
@@ -0,0 +1,23 @@
+from .sensor_definition import Sensor
+from time import sleep
+from picamera import PiCamera
+from datetime import datetime
+
+class PiCam(Sensor):
+
+	def __init__(self) -> None:
+		self.camera = PiCamera()
+		self.camera.rotation = 180
+
+	def read_value(self):
+		now = datetime.now()
+		date_str = now.strftime("%m_%d_%Y__%H:%M")
+		self.camera.start_preview()
+		self.camera.annotate_text = date_str
+		sleep(3)
+		self.camera.capture('/home/pi/Documents/Plant_Photos/%s.jpg' % date_str)
```

Comment:
Should we let this be configurable too? From `__init__`?

Reply:
yeah I think that would be better, let me change that